OpenSim/Simulation/Model/Model.h

```diff
@@ -761,7 +768,12 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Model, ModelComponent);
     SimTK::Vec3 calcMassCenterPosition(const SimTK::State &s) const;
     SimTK::Vec3 calcMassCenterVelocity(const SimTK::State &s) const;
     SimTK::Vec3 calcMassCenterAcceleration(const SimTK::State &s) const;
-
+    double calcKineticEnergy(const SimTK::State &s) const {
+        return getMultibodySystem().calcKineticEnergy(s);
+    }    
+    double calcPotentialEnergy(const SimTK::State &s) const {
+        return getMultibodySystem().calcPotentialEnergy(s);
+    }
```

Comment:
If you're planning on adding more changes to this branch anyway, could you add comments here simply saying that these quantities are for the total system?